<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Toiminnot</class>
 <widget class="QDialog" name="Toiminnot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButtonNosto">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Tee nosto</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSaldo">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Näytä saldo</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Saldosi on</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonTakaisin">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Takaisin</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSumma">
   <property name="geometry">
    <rect>
     <x>212</x>
     <y>140</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>120</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Syötä nostettava summa</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOk">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>170</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonTakaisin2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>170</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Takaisin</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSaldo">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelSaldoWithdraw">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>210</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Saldosi noston jälkeen</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelUusiSaldoWithdraw">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>210</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
